File: back-end/src/main/java/io/enfuse/rpsgame/GameScoreResponse.java
package io.enfuse.rpsgame;

public class GameScoreResponse {

    private int ties, firstPlayerWins, secondPlayerWins;

    public GameScoreResponse() {
        this.ties = 0;
        this.firstPlayerWins = 0;
        this.secondPlayerWins = 0;
    }

    public GameScoreResponse(int ties, int firstPlayerWins, int secondPlayerWins) {
        this.ties = ties;
        this.firstPlayerWins = firstPlayerWins;
        this.secondPlayerWins = secondPlayerWins;
    }

    public int getTies() {
        return ties;
    }

    public int getFirstPlayerWins() {
        return firstPlayerWins;
    }

    public int getSecondPlayerWins() {
        return secondPlayerWins;
    }
}

File: back-end/rps-front-end/src/domain/game-score-response.ts
export class GameScoreResponse {
    ties:number;
    firstPlayerWins:number;
    secondPlayerWins:number;

    constructor(ties : number, firstPlayerWins : number, seconPlayerWins :number){
        this.ties = ties;
        this.firstPlayerWins = firstPlayerWins;
        this.secondPlayerWins = seconPlayerWins;
    }
}